sdk/typescript/test/e2e/object-vector.test.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, it, expect, beforeAll } from 'vitest';
+import {
+  Base64DataBuffer,
+  getCreatedObjects,
+  getEvents,
+  getExecutionStatusType,
+  LocalTxnDataSerializer,
+  ObjectId,
+  RawSigner,
+} from '../../src';
+import { DEFAULT_GAS_BUDGET, setup, TestToolbox } from './utils/setup';
+// TODO: this file is generated through
+// `sui move build --dump-bytecode-as-base64 --path sui/crates/sui-core/src/unit_tests/data/entry_point_vector`
+// This is fragile but needed for now because we don't know the path of user's Sui CLI in this e2e test. A
```

Comment:
You should be able to use cargo to invoke the sui CLI?

Reply:
Good point! Rewrite the logic to use `cargo run` instead. But I noticed that it takes around ~5 seconds to run the `cargo run` command(even when the binary has been built), so we may still want to add the `move build` endpoint to the `sui-test-validator`

Reply:
Yeah I think the test validator could hold this in the future for sure!